### ISSUE TITLE
When will there be a 1.3 tag? (currently 1.2 fails on FreeBSD)

### ISSUE THREAD

**Issue by hxw** [berdario/pew#232]
The problem is that FreeBSD Python does not support the code containing `import distutils; ` to access distutils.sysconfig for the Python library path. I see that this code has been changed by https://github.com/berdario/pew/commit/cf6c2204750d17a0bf2974df4d3263c20ef3ca1d

Manually applying the patch allows pew to function.

Since FreeBSD Python ports all use PyPi for source downloads, it would need a new tag and new upload to PyPi to be able to fix this properly. Thanks.

**Comment by hxw** [comment]
Hello @tjanez do you know if there is any plan to tag a newer version and update PyPi?

**Comment by tjanez** [comment]
> I think @berdario is the only person who can do a new release,

Yes, I don't have access to Pew's PyPI so even if I could do a new tag on GitHub, I couldn't do a release on PyPI.

> and he's not been very active here in quite a while. So I don't know if a new release is likely any time soon.

Agreed...
